pull

```diff
@@ -66,4 +66,10 @@ cd ./ffc-sfd-customer-receiver-payments && \
   git pull
 cd "${projectRoot}"
 cd ./ffc-sfd-customer-receiver-preferences && \
+  git pull
+cd "${projectRoot}"
+cd ./ffc-sfd-customer-receiver-queries && \
+  git pull
+cd "${projectRoot}"
+cd ./ffc-sfd-queries && \
   git pull
```

Comment:
Made updates to the `README` plus the `open`, `build`, and `pull` scripts so that repos are grouped based on whether they're frontends, backends, or receivers.